src/components/signup-confirmation/SignupConfirmation.less

```diff
@@ -0,0 +1,48 @@
+.input-style {
```

Comment:
less files are in kebab case however! (why? I have no idea, but its standard apparently).
So signup-confirmation.less